### ISSUE TITLE
Kong API url is calculated based on Kong Admin url

### ISSUE THREAD

**Issue by maksimlikharev** [Haufe-Lexware/wicked.node-sdk#4]
```
function _getInternalKongAdminUrl() {
    debug('getInternalKongAdminUrl()');
    checkInitialized('getInternalKongAdminUrl');
    return _getInternalUrl('kongAdminUrl', 'kong', 8001);
}
exports._getInternalKongAdminUrl = _getInternalKongAdminUrl;
/** @hidden */
function _getInternalKongProxyUrl() {
    debug('getInternalKongAdminUrl()');
    checkInitialized('getInternalKongAdminUrl');
    // Deduce from Kong Admin URL
    const adminUrl = _getInternalKongAdminUrl();
    return adminUrl.replace(/8001/, '8000');
}
exports._getInternalKongProxyUrl = _getInternalKongProxyUrl;
```

**return adminUrl.replace(/8001/, '8000');**

This is really incorrect logic, as api URL, if done correctly will be different from admin URL, as admin port is privileged  and should be protected.
We have following load balancers:

```
kong-admin.xxx

kong-api.xxx

```

this really steams from investigation of why getAccessToken() fails for me, as it uses that as internal api url.

It should be overwritten with a constant and env. variable the same way as kongAdminUrl

**Comment by DonMartin76** [comment]
Aha, okay. I get it. For Kubernetes/docker deployments this is absolutely identical, but in your case it does matter.

I would do it like this: I would open up an additional field in the globals.network settings (and wire it in the Kickstarter), like "internal Kong Proxy URL" which you *may* use, but don't have to use (it would fall back to this behaviour).

Sounds fair?

**Comment by maksimlikharev** [comment]
yes, as long as I can set it to a different value. 
thank you.

**Comment by DonMartin76** [comment]
<img width="1197" alt="image" src="https://user-images.githubusercontent.com/7326883/46128636-7150aa80-c234-11e8-946a-d8481fcede9b.png">

Is this fine?

**Comment by DonMartin76** [comment]
Code:

```
export function _getInternalKongProxyUrl() {
    debug('getInternalKongProxyUrl()');
    checkInitialized('getInternalKongProxyUrl');

    // Check if it's there
    try {
        const proxyUrl = _getInternalUrl('kongProxyUrl', 'kong', 8000);
        if (proxyUrl)
            return proxyUrl;
    } catch (ex) {
        debug(ex);
    }
    debug(`globals.json: network.kongProxyUrl is not defined, falling back to admin URL.`)
    // Fallback: Deduce from Kong Admin URL
    const adminUrl = _getInternalKongAdminUrl();
    return adminUrl.replace(/8001/, '8000');
}

```

**Comment by DonMartin76** [comment]
Will be fixed/supported in 1.0.0.beta9.
